projects/demo/src/modules/components/pull-to-refresh/examples/3/index.html

```diff
@@ -4,7 +4,8 @@
             appendOnly
             itemSize="50"
             tuiScrollable
-            class="example-viewport tui-zero-scrollbar"
+            class="example-viewport"
+            [class.tui-zero-scrollbar]="!nativeScrollbar"
```

Comment:
Please remove this from all demo pages — it is irrelevant to the demo.

Reply:
fixed